modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -95,7 +119,9 @@ exports.changeProfilePicture = function (req, res) {
 
   function updateUser() {
     return new Promise(function (resolve, reject) {
-      user.profileImageURL = config.uploads.profile.image.dest + req.file.filename;
+      user.profileImageURL = config.uploads.storage === 's3' && config.aws.s3 ?
+        req.file.location :
+        '/' + req.file.path;
```

Comment:
This can be changed back to what it was before: `config.uploads.profile.image.dest + req.file.filename`, if we can still use the "/" on the front-end when using local storage.

Reply:
I'm not fan of the '/' in the front end. I prefer having the final URL in the database. Otherwise if we switch from local to s3, all old URL would be bad formed. And that add more logic to the front where it shouldn't be.